public/js/filterbar.js

```diff
@@ -34,18 +33,18 @@ $(function() {
 				}
 				return true;
 			}
-		}).show();
-
+		});
+		filteredRows.show();
+		var emptySearch = $('.empty-search');
+		filteredRows.length === 0 ? emptySearch.attr('aria-hidden', 'false') : emptySearch.attr('aria-hidden', 'true');
 		return rows;
 	}
 
 	function navigateRows(key) {
-
-
-		var rows = $componentList.find(itemSelector + ':visible'),
-			current = rows.filter('.focused'),
-			index = rows.index(current),
-			select = 0;
+		var rows = $componentList.find(itemSelector + ':visible');
+		var	current = rows.filter('.focused');
```

Comment:
Yes, linting was failing because of the declaration of multiple variables at once, that is why I changed it. I'll fix now the whitespace.